synapse/events/third_party_rules.py

```diff
@@ -353,6 +365,50 @@ async def on_new_event(self, event_id: str) -> None:
                     "Failed to run module API callback %s: %s", callback, e
                 )
 
+    async def check_can_shutdown_room(self, user_id: str, room_id: str) -> bool:
+        """Intercept requests to shutdown a room. If `False` is returned, the
+        room should not be shut down.
```

Comment:
```suggestion
        room must not be shut down.
```

Since this matches reality better